api/types/github.go

```diff
@@ -235,5 +259,18 @@ func (c *GithubConnectorV3) MapClaims(claims GithubClaims) ([]string, []string,
 			}
 		}
 	}
-	return utils.Deduplicate(logins), utils.Deduplicate(kubeGroups), utils.Deduplicate(kubeUsers)
+	for _, mapping := range c.GetTeamsToRoles() {
+		teams, ok := claims.OrganizationToTeams[mapping.Organization]
+		if !ok {
+			// the user does not belong to this organization
+			continue
+		}
+		for _, team := range teams {
+			// see if the user belongs to this team
+			if team == mapping.Team {
+				roles = append(roles, mapping.Roles...)
```

Comment:
What about adding kubeGroups and kubeUsers from the mapping like above?

Reply:
The new mapping doesn't have these, if we have roles, I don't see a need for them. They were there because this mapping was originally introduced before OSS had RBAC.

Reply:
Ah yes, you're right.